dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -724,6 +724,14 @@
 				status = "disabled";
 			};
 		};
+
+		sdmmc1: sdmmc@52007000 {
```

Comment:
H7 series have `sdmmc1` and `sdmmc2`, I think both should be added.

Reply:
I can't test the other one, prefer to add things I got working.

Reply:
Added